docs/source/coregistration.rst

```diff
@@ -1,7 +1,16 @@
 .. _coregistration:
 
-DEM Coregistration
-==================
+Coregistration
+===============
+
+Coregistration between DEMs correspond to aligning the digital elevation models in three dimension.
+
+Transformations that can be described by a 3-dimensional affine function are included in coregistration methods.
+Those transformations include :
+
+- vertical and horizontal shifts,
+- rotations,
+- stretching and squeezing.
```

Comment:
I would call it "scale change". For me squeezing sounds like np.squeeze, that we're seeing way to often in xdem! ;-)

Reply:
But it's not always just scaling! It's warping too with Deramp and the upcoming BlockwiseCoreg

Reply:
And scaling is the commonly used term: https://en.wikipedia.org/wiki/Affine_transformation

Reply:
Good point. Maybe we should have scaling, and then another point that is e.g. "non-linear shifts" or so

Reply:
Or "rubber-sheet" deformation (or whatever you called it @erikmannerfelt at some point.